app/src/services/api/urls.ts

```diff
@@ -1 +1,8 @@
-export const { REACT_APP_API_URL: API_URL } = process.env;
+let API_URL = "" as string | undefined;
```

Comment:
this is an issue for me: it disturbs the development workflow. IMO we want to be able to use a `.env` during development and keep using `process.env`: we keep templating for production only. Otherwise, `yarn start` and code reloading won't work anymore.

Reply:
tried your suggestion, I like it after all